crates/diesel_utils/replaceable_schema/triggers.sql

```diff
@@ -696,3 +697,145 @@ CREATE TRIGGER search_combined_community_score
     AFTER UPDATE OF users_active_month ON community
     FOR EACH ROW
     EXECUTE FUNCTION r.search_combined_community_score_update ();
+-- Multi_community score
+CREATE FUNCTION r.search_combined_multi_community_score_update ()
+    RETURNS TRIGGER
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+    UPDATE
+        search_combined
+    SET
+        score = NEW.subscribers
+    WHERE
+        multi_community_id = NEW.id;
+    RETURN NULL;
+END
+$$;
+CREATE TRIGGER search_combined_multi_community_score
+    AFTER UPDATE OF subscribers ON multi_community
+    FOR EACH ROW
+    EXECUTE FUNCTION r.search_combined_multi_community_score_update ();
+-- Increment / decrement multi_community counts
+CREATE FUNCTION r.multicommunity_community_increment ()
+    RETURNS TRIGGER
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+    UPDATE
+        multi_community
+    SET
+        communities = communities + 1
+    WHERE
+        id = NEW.multi_community_id;
+    RETURN NULL;
+END
+$$;
+CREATE FUNCTION r.multicommunity_community_decrement ()
+    RETURNS TRIGGER
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+    UPDATE
+        multi_community
+    SET
+        communities = communities - 1
+    WHERE
+        id = OLD.multi_community_id;
+    RETURN NULL;
+END
+$$;
+CREATE TRIGGER multi_community_add_community
+    AFTER INSERT ON multi_community_entry
+    FOR EACH ROW
+    EXECUTE FUNCTION r.multicommunity_community_increment ();
+CREATE TRIGGER multi_community_remove_community
+    AFTER DELETE ON multi_community_entry
+    FOR EACH ROW
+    EXECUTE FUNCTION r.multicommunity_community_decrement ();
+-- Increment / decrement multi_community subscriber counts
+CREATE FUNCTION r.multicommunity_subscribers_increment ()
+    RETURNS TRIGGER
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+    UPDATE
+        multi_community
+    SET
+        subscribers = subscribers + 1
+    WHERE
+        id = NEW.multi_community_id;
+    RETURN NULL;
+END
+$$;
+CREATE FUNCTION r.multicommunity_subscribers_decrement ()
+    RETURNS TRIGGER
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+    UPDATE
+        multi_community
+    SET
+        subscribers = subscribers - 1
+    WHERE
+        id = OLD.multi_community_id;
+    RETURN NULL;
+END
+$$;
+CREATE TRIGGER multi_community_add_subscribers
+    AFTER INSERT OR UPDATE OF follow_state ON multi_community_follow
```

Comment:
This doesn't correctly handle updates other than changing follow_state to accepted. It should be fixed by either adding a separate update trigger that checks `(OLD.follow_state, NEW.follow_state)` or using the trigger style that's used for community follows.

Reply:
`WHEN (NEW.follow_state = 'Accepted')` should handle that, right? What case is missing?

Reply:
Thx, I think I got it now. These all might not be necessary, since a lot of the updates like changing from accepted -> pending are probably never possible, but it doesn't hurt.